src/domain/map/components/HankeSidebar/HankeSidebar.tsx

```diff
@@ -63,16 +127,43 @@ const HankeSidebar: React.FC<React.PropsWithChildren<Props>> = ({ hanke, isOpen,
             <IconCross aria-hidden />
           </button>
           <Text tag="h2" weight="bold" styleAs="h4" spacing="2-xs">
-            {hanke.nimi} ({hanke.hankeTunnus})
-          </Text>
-          {hanke.tyomaaKatuosoite && (
-            <Text tag="p" styleAs="h5" weight="bold" spacingBottom="2-xs">
-              {hanke.tyomaaKatuosoite}
-            </Text>
-          )}
-          <Text tag="p" styleAs="h6" weight="bold" spacingBottom="s">
-            {formatToFinnishDate(hanke.alkuPvm)} - {formatToFinnishDate(hanke.loppuPvm)}
+            {hanke.nimi}: {hankealue.nimi}
           </Text>
+          <SidebarSection
+            title={t('hankeSidebar:labels:hankealueenAjankohta')}
+            content={`${formatToFinnishDate(hankealue.haittaAlkuPvm)} - ${formatToFinnishDate(hankealue.haittaLoppuPvm)}`}
+          />
+          <SidebarSection
+            title={t('hankeSidebar:labels:hanke')}
+            content={
+              isUsersHanke ? (
+                <Link
```

Comment:
Could `react-router` `Link` be used here as it will not reload the whole page.

Reply:
I changed it to use Link from react-router